docs/api/drivers/driver_interfaces.md

```diff
@@ -103,7 +103,7 @@ The inheritance of a driver from the new driver interface is conditional upon `F
 
 When `FEATURE_EXPERIMENTAL_API` is disabled, the following type alias is made instead:
 
-```c++
```

Comment:
not sure why this is being removed here, as the other code snippets in this file are marked as being C++ in the same way.

Reply:
yeah, this failed at  completion job,  as this code already part of mbed-os code, so compile this snippet will through out error.  unmark this as `c++` will suppress the  compile test for this snippet

Reply:
ok (the website doesn't seem to do any useful syntax highlighting based on this anyway)